packages/is-valid-titulo-eleitor/src/index.js

```diff
@@ -0,0 +1,26 @@
+import onlyNumbers from '@brazilian-utils/helper-only-numbers';
+
+import { BLACKLIST, TITULO_ELEITOR_LENGTH } from './constants';
+
+const isValidLength = tituloEleitor => tituloEleitor.length === TITULO_ELEITOR_LENGTH;
+
+const belongsToBlacklist = tituloEleitor => BLACKLIST.includes(tituloEleitor);
+
+const modSum = (arr, weight) => arr.reduce((acc, digit) => weight < 10 ? acc + digit * weight++ : 0, 0) % 11;
+
+const isValid = (arr, mod1, mod2) => (arr[10] === mod1 && arr[11] === mod2) && ((arr[8] + arr[9] > 0) && (arr[8] + arr[9] < 29));
```

Comment:
I don't know if `arr`, `mod1` and `mod2` are better names for these variables and I didn't understand what's happening inside this function, maybe you can describe with variables what happen. For example: `arr[10]` to `firstCheckDigit`.

Reply:
I have a feeling that are too much of magic here.

There are lots of literal integers that we can name to make the code more readable.